src/utils/clack-utils.ts

```diff
@@ -614,7 +616,16 @@ SENTRY_AUTH_TOKEN=${authToken}
 }
 
 async function addCliConfigFileToGitIgnore(filename: string): Promise<void> {
-  //TODO: Add a check to see if the file is already ignored in .gitignore
+  const regex = new RegExp(`^\\s*${filename}\\s*=`, 'g');
```

Comment:
Two questions:
1. why do we need the final `=`?
2. should this have `gm` for flags? 

Couldn't get the regex to match on a simple input without changing 1 and 2 (https://regex101.com/r/0I5uLE/1).

Reply:
Thanks for the help @Lms24, much appreciated! I didn’t test this regex before committing and that is definitely my bad!

It was copied wholesale from the regex for SENTRY_AUTH_TOKEN in src/utils/clack-utils.ts

Reply:
No worries! The auth token itself isn't added to `.gitignore` but to an env file, hence the `=` is in there. No rush, feel free to update whenever you have time :)

Reply:
@Lms24 Updated now! 😅